platform_umbrella/apps/common_core/lib/common_core/resources/sso/keycloak.ex

```diff
@@ -40,7 +40,7 @@ defmodule CommonCore.Resources.Keycloak do
       # Make sure Keycloak uses the headless service for DNS resolution
       # which is how it does clustering
       |> Map.put("jgroups.dns.query", "keycloak-headless.#{namespace}")
-      |> Map.put("KC_CACHE_STACK", "kubernetes")
+      |> Map.put("KC_CACHE_STACK", "jdbc-ping")
       # Metrics and Health enabled
       |> Map.put("KC_METRICS_ENABLED", "true")
       |> Map.put("KC_HEALTH_ENABLED", "true")
```

Comment:
Should we go ahead and make some of these parts of the config? A defaultable field for some of the more important settings would be nice for testing if these work.